gradle/web.gradle

```diff
@@ -129,7 +154,7 @@ task web(type: Exec) {
     }
   }
 
-  def sleepUntilDockerExits = "while docker ps --format '{{.Names}}' | grep -q '${projectName}-${subProjectName}'; do echo -e \"\\r\\033[1A\\033[0K[\$(date +'%Y-%m-%d %T')] waiting for [\$(docker ps --format '{{.Names}}' | grep '${projectName}-${subProjectName}' | wc -l | tr -d '\t')] ${projectName}-${subProjectName} running containers to finish...\" && sleep 5; done"
+  def sleepUntilDockerExits = "while docker ps --format '{{.Names}}' | grep -q '${projectName}-${subProjectName}-web'; do echo -e \"\\r\\033[1A\\033[0K[\$(date +'%Y-%m-%d %T')] waiting for [\$(docker ps --format '{{.Names}}' | grep '${projectName}-${subProjectName}' | wc -l | tr -d '\t')] *${projectName}-${subProjectName}-web* running containers to finish...\" && sleep 2; done"
```

Comment:
Is this "-web" the same as the prefix? Should it be a variable as well?

Reply:
Just to denote that it is a web container as opposed to others that may be generated by the service that aren't web related.